src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1429,85 +1228,77 @@ class SaagieClient {
 
-                        if(configuration.pipeline.include_all_versions) {
-                            def listInstancesPipelines = getAllInstancePipelineInformation(pipelineId)
-                           def mappedListInstancesPipelines = listInstancesPipelines.collect{ workflow ->
-                                getInstancePipelineDetail(workflow.workflowId as String, workflow.id as String)
-                            }
-                           mappedListInstancesPipelines.collect { workflow ->
-                                workflow.jobs = workflow.jobs.id
-                                return workflow
-                           }
-                            mappedListInstancesPipelines.forEach{
-                                exportPipeline.addPipelineVersionDTOtoVersions(it.jobs, null)
-                            }
-                           if(exportPipeline.versions.size() > 1) {
-                               exportPipeline.versions.unique()
-                               exportPipeline.versions.collect { workflow ->
-                                   workflow.jobs = workflow.jobs.collect{ job->
-                                       [id: job]
-                                   }
-                               }
-                           }
+                        mappedListInstancesPipelines.collect { workflow ->
+                            workflow.jobs = workflow.jobs.id
+                            return workflow
+                        }
 
+                        mappedListInstancesPipelines.forEach {
+                            exportPipeline.addPipelineVersionDTOtoVersions(it.jobs, null)
                         }
-                        exportPipeline.setPipelineVersionFromV1ApiResult(
-                            parsedV1PipelineResult.jobs.collect{ job ->
-                                [id: job.id]
-                            },
-                            parsedV1PipelineResult.hasProperty("releaseNote") ? parsedV1PipelineResult.releaseNote : null
-                        )
-                   }
 
-                return exportPipeline
-            }
+                        if(exportPipeline.versions.size() > 1) {
+                            exportPipeline.versions.unique()
+                            exportPipeline.versions.collect { workflow ->
+                                workflow.jobs = workflow.jobs.collect{ job ->
+                                    [id: job]
+                                }
+                            }
+                        }
+                    }
 
+                    exportPipeline.setPipelineVersionFromV1ApiResult(
+                        parsedV1PipelineResult.jobs.collect{ job ->
+                            [id: job.id]
+                        },
+                        parsedV1PipelineResult.hasProperty("releaseNote") ? parsedV1PipelineResult.releaseNote : null
```

Comment:
```suggestion
                        parsedV1PipelineResult?.releaseNote
```

You can see how it's working here: https://groovy-lang.org/operators.html#_safe_navigation_operator ;)